### PR TITLE
[WIP] Ran futurize for fun

### PR DESCRIPTION
**Pull request by kyleabeauchamp**

### CODE REVIEW

File: stor/base.py [hunk 1]
```diff
@@ -388,12 +392,12 @@ def listdir(self):
 
         The elements of the list are Path objects.
         """
-        return [self / child
+        return [old_div(self, child)
                 for child in map(self._always_unicode, os.listdir(self))]
 
     def glob(self, pattern):
         cls = self.path_class
-        return [cls(s) for s in glob.glob(self / pattern)]
```
Comment (kyleabeauchamp) [inline stor/base.py]: this one is pretty funny.

Comment (kyleabeauchamp) [inline stor/base.py]: Presumably we should disable that flag and do division changes manually given the abundance of string division.

File: stor/obs.py [hunk 2]
```diff
@@ -276,9 +280,9 @@ def __iter__(self):
     def _buffer(self):
         "Cached buffer of data read from or to be written to Object Storage"
         if self.mode in ('r', 'rb'):
-            return cStringIO.StringIO(self._path.read_object())
+            return io.StringIO(self._path.read_object())
```
Comment (lucaswiman) [inline stor/obs.py]: I think these should be `BytesIO`. Object Storage is storing binary blobs of data, not unicode.

Comment (kyleabeauchamp) [inline stor/obs.py]: Indeed, good to see failure modes of automatic conversion :)

**Comment by kyleabeauchamp** [comment]
Deprecated by #10

**Pull request closed without merging**